AEP/aep-00001.md

```diff
@@ -0,0 +1,177 @@
+**Author**| Dylan Brasseur <<dbrasseur@aneo.fr>>
+**Status**| Draft
+**Type**| Process
+
+# What is an AEP ?
+
+AEP stands for ArmoniK Enhancement Proposal. An AEP is a design document providing information to the ArmoniK community, or describing a new feature for ArmoniK, its processes or environment. The AEP should provide a concise technical specification of the feature and a rationale for the feature.
+
+We intend AEPs to be the primary mechanisms for proposing major new features, for collecting community input on an issue, and for documenting the design decisions that have gone into ArmoniK. The AEP author is responsible for building consensus within the community and documenting dissenting opinions.
+
+Because the AEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal. This historical record is available by the normal git commands for retrieving older revisions, and can also be browsed [on GitHub](https://github.com/aneoconsulting/ArmoniK.Community).
+
+# AEP Audience
+
+The typical primary audience for AEPs is the ArmoniK Developer Team.
+
+However, other parts of the ArmoniK community may also choose to use the process (particularly for Informational AEPs) to document expected API conventions and to manage complex design coordination problems that require collaboration across multiple projects. External contributors should also follow the guidelines established by the AEPs.
+
+# AEP Types
+
+There are three kinds of AEP:
+
+1. A **Standard** AEP describes a new feature or implementation for ArmoniK.
+2. An **Informational** AEP describes an ArmoniK design issue, or provides general guidelines or information to the ArmoniK community, but does not propose a new feature. Informational AEPs do not necessarily represent an ArmoniK community consensus or recommendation, so users and implementers are free to ignore Informational AEPs or follow their advice.
+3. A **Process** AEP describes a process surrounding ArmoniK, or proposes a change to (or an event in) a process. Process AEPs are like Standard PEPs but apply to areas other than ArmoniK itself. They may propose an implementation, but not to ArmoniK’s codebase; they often require community consensus; unlike Informational AEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in ArmoniK development. Any meta-AEP is also considered a Process AEP.
+
+# AEP Workflow
+
+## ArmoniK Developer Team
+
+In AEPs, the "ArmoniK Developer Team" refers to the team at [Aneo](https://www.aneo.eu/) in charge of deciding the core functionnalities and direction of ArmoniK. This term does not extend to ArmoniK external contributors. The ArmoniK Developer Team decides whether a proposal should be considered as an AEP and they usually develop the feature described in them.
```

Comment:
Je pense que nous devrions mettre la liste des membres Core dans une AEP.

Reply:
I wholefully agree. Should we already reserve an AEP number for it and add it to this AEP ?

Reply:
I think so (the 0005?)